content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -68,20 +70,34 @@ Typically, on the scanner, a carriage return is included by default. Ensure it i
 specific barcode in the user manual, like `CR suffix ON` or `Apply Enter for suffix`.
 
 Zebra scanner
--------------
+=============
 
 When using Zebra scanners, ensure the following keystroke configurations are set to prevent errors.
 
 Begin on the Zebra scanner's home screen, and select the :guilabel:`DataWedge` app (the icon for the
 app is a light blue barcode). On the :guilabel:`DataWedge Profiles` page, select the profile option
 to access the Zebra scanner's settings.
 
-Scroll down to the :guilabel:`Keyboard Output` option, and ensure the :guilabel:`Enable/disable
-keystroke output` option is :guilabel:`Enabled`.
+.. warning::
+   It is not suggested to use the "DWDemo" profile, as it does not work properly in every
+   circumstance. Instead, it is recommended to create a new, personal profile. Once a new profile is
+   created, add the Odoo Mobile App and Google Chrome in the *Associated Apps* on the scanner's home
+   screen.
```

Comment:
Do you have a Zebra scanner on-hand to test? I took a look at the ticket that Darryl linked, and there are a little bit more details on how to add Odoo Mobile and Google Chrome. 
![image](https://github.com/odoo/documentation/assets/23611711/b30713ff-f831-4f2e-b307-7c8fb499bd42)

Adding the specific instructions might make this important admonition block more bulky, but I think it's always good to be very specific about instructions, especially since users don't have resources other than our documentation. 
 
If you have a zebra scanner that you can test with, I'd recommend going through the instructions provided by jelu, the support agent, and naming the dropdown menus, headings, and buttons, so that readers are super clear on where to go to add Chrome and Odoo. If you don't have access to a scanner, I can get one from the SF office and I can fill in the details for you! Just let me know (: